data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/chance_to_hit/chance_to_hit_wfl_other.cfg

```diff
@@ -0,0 +1,39 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [chance_to_hit]value=
+##
+# Actions:
+# Give the leaders a chance_to_hit ability that gives a quarter of the max hp of the other unit as strikes.
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# chance_to_hit has a value of 100 (29+71)
```

Comment:
A couple of `[have_unit]` asserts would be nice, just to sanity-check that this assumption hasn't been changed by an orc rebalance.

Reply:
What would this affect? No unit involved has any chance of dying.

Reply:
It would affect the chance to hit...

Reply:
Updated. ASSERT didn't work for some reason, so I reused ASSERT_UNIT_HP_XP.